news/2024/2024-05-01-the-nat-gazette-june-2024.md

```diff
@@ -0,0 +1,124 @@
+
+#### Streamlining cooldowns
+
+Cooldowns have been homogenized across the board. Failed applicants and BNs who get kicked for low activity will always have to wait 60 days before reapplying again under normal circumstances. This does not apply to Beatmap Nominators who leave on good terms and want to rejoin within a year after their departure who will be able to rejoin instantly.
+
+#### Progress bar for applications
+
+One of the issues with the opaqueness of the process was that candidates didn't know when their verdict would be delivered, or at what stage their application was. By offering our very own BN app parcel tracking service, applicants will now be able to be updated in real time when their food — I mean, their application — be delivered.
+
+Insert image perchance.
+
+#### Reworking application feedback
+
+Feedback has been simplified. In short, this change will make the feedback writing process which was deemed redundant be less time-consuming. This will quicken the time for the results of an application to be delivered. The evaluators' comments will instead serve as more in depth feedback.
+
+#### Communication platform post-applications (need a better title) (I couldn't come up with one either)
+
+After receiving their verdict, users can reply to their evaluators' feedback in the BN site. While the goal of this change is not to facilitate appeals as under normal circumstances the result of an application is unchangeable, applicants are encouraged to ask questions and clarification on their feedback by using this new feature.
+
+### Within the Beatmap Nominators
+
+#### Probation is no longer a demotion option
+
+The Probation Beatmap Nominators user group has been reworked slightly. It will now serve its original goal, which is as a trial ground for green beards! When a new Beatmap Nominator is allowed into the team, they will be put in probation for their first couple months. After evaluating their performance with actual BN work, they will either be promoted or removed. Full Beatmap Nominators cannot under any circumstance be demoted back into probation.
+
+#### Mock evaluations are no more
+
+Beatmap Nominators will not be able to make mock evaluations anymore. The system besides being underused, was usually superseded by the introduction of BN Evaluator cycles.
+
+#### Vetoes
+
+Vetoes have slightly been reworked. Once a Veto is made, the entire usergroup of the gamemode which it concerns will be drafted. While participation is not obligatory, this change aims to provide a better representation of what the BN group of a gamemode thinks of a Veto, in turn making the results feel less RNG depending on who got randomly chosen.
```

Comment:
```suggestion
Vetoes have been slightly reworked. Once a veto is made, the entire user group of the game mode which it concerns will be drafted instead of a random jury. While participation is not obligatory, this change aims to provide a better representation of what the BN group of a game mode thinks of a veto, in turn making the results feel less like RNG.
```